cmd/call-endpoint/main.go

```diff
@@ -76,3 +84,14 @@ func main() {
 
 	cmd.MustHandleCMDRequestAndExitFull(args.Endpoint, request, nil, args.CommonOptions, printFunc)
 }
+
+func listAPIEndpoints() {
+	apiDescription, err := api.Describe(*api.GetRoutes())
+	if err != nil {
+		log.Fatal("Failed to describe API endpoints: '%w'.", err)
+	}
+
+	for endpoint := range apiDescription.Endpoints {
```

Comment:
I don't like one-valued map range (I don't think it is as clear as using an empty value).
Put in an underscore for the value.